src/stores/active-media.js

```diff
@@ -0,0 +1,68 @@
+import { defineStore } from 'pinia'
+import { reactive, readonly, toRefs } from '@nuxtjs/composition-api'
+
+const ACTIVE_MEDIA = 'active-media'
+
+/**
+ * Store information about the active media item.
+ */
+export const useActiveMediaStore = defineStore(ACTIVE_MEDIA, () => {
+  /**
+   * `reactive` returns UnwrapRef<T> type, but the Vue docs recommend using the
+   * type of <T> for typing it instead:
+   * https://vuejs.org/guide/typescript/composition-api.html#typing-reactive
+   * @type {import('../store/types').ActiveMediaState}
+   */
+  const state = reactive({
+    type: null,
+    id: null,
+    status: 'ejected',
+    message: null,
+  })
+  /**
+   * Only the properties used by components are exported as refs.
+   * `status` is not used anywhere in the components.
+   */
+  const { type, id, message } = toRefs(state)
```

Comment:
I couldn't find any usages of `activeMediaStore.type`. `id` and `message` both have usages though.

Reply:
I think `type` will be used when we have video, and we can have an `activeVideo` which will need to be treated differently from `activeAudio`